students/Gorthi_kavita/10module/10assignement.py

```diff
@@ -0,0 +1,173 @@
+class donar():
+    list =[]   #this is donar list adding  to the list
+    allist =[]   #to keep track how many times the donar name in the list
+    uniquename = []  # unique donar names
+    listsum = []     # donar names and total amount and number of times donated
+    listadd =[]
+    finalsum = []
+
+    def __data(self,name,amount):
+        self.donarname = name
+        self.donaramount = amount
+        c = (name,amount,1)
```

Comment:
avoid single character variable names.  Use full words whenever possible. Refer to https://www.python.org/dev/peps/pep-0008/ for the python style guide.